src/MongoDB/ServerDescription.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright 2021-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <php.h>
+#include <Zend/zend_interfaces.h>
+#include <ext/standard/php_var.h>
+#include <zend_smart_str.h>
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "phongo_compat.h"
+#include "php_phongo.h"
+
+zend_class_entry* php_phongo_serverdescription_ce;
+
+/* {{{ proto array MongoDB\Driver\ServerDescription::getHelloResponse()
+   Returns the most recent "hello" response */
+static PHP_METHOD(ServerDescription, getHelloResponse)
+{
+	php_phongo_serverdescription_t* intern;
+	const bson_t*                   helloResponse;
+
+	intern = Z_SERVERDESCRIPTION_OBJ_P(getThis());
+
+	PHONGO_PARSE_PARAMETERS_NONE();
+
+	helloResponse = mongoc_server_description_hello_response(intern->server_description);
+
+	php_phongo_bson_state state;
+
+	PHONGO_BSON_INIT_DEBUG_STATE(state);
+
+	if (!php_phongo_bson_to_zval_ex(bson_get_data(helloResponse), helloResponse->len, &state)) {
+		/* Exception should already have been thrown */
+		zval_ptr_dtor(&state.zchild);
+		return;
+	}
+
+	RETURN_ZVAL(&state.zchild, 0, 1);
+} /* }}} */
+
+/* {{{ proto string MongoDB\Driver\ServerDescription::getHost()
+   Returns the server’s hostname */
```

Comment:
Didn't realize these were curly quotes as well.

```suggestion
   Returns the server's hostname */
```